hail_search/hail_search_query.py

```diff
@@ -59,6 +63,14 @@ class BaseHailTableQuery(object):
         XPOS: lambda r: [r.xpos],
     }
 
+    @classmethod
+    def load_globals(cls):
+        cls.LOADED_GLOBALS = {}
+        for genome_version in cls.GENOME_VERSIONS:
+            ht_path = cls._get_generic_table_path(genome_version, 'annotations.ht')
+            ht_globals = hl.eval(hl.read_table(ht_path).globals.select(*cls.GLOBALS))
+            cls.LOADED_GLOBALS[genome_version] = {k: ht_globals[k] for k in cls.GLOBALS}
```

Comment:
Does it necessary to load multiple genome versions? It looks that only the version of `self._genome_version` is useful.

Reply:
yes because we have data on both builds so we need to support both builds, so when users do searches on either build they work. we different reference data sources for different data builds, so the enums could easily be different